_posts/2019-08-06-serverless-static-sites.md

```diff
@@ -0,0 +1,232 @@
+After a while, if you head over to your personal dashboard(find it at `system.<your domain>/dashboard/<username>`) you will see your new function deployed.  
+Edit the `FunctionIngress` so that it points to our brand new function `kubectl edit -n openfaas functioningresses.openfaas.com example-site`:
+```yaml
+apiVersion: openfaas.com/v1alpha2
+kind: FunctionIngress
+metadata:
+  name: example-site-tls
+  namespace: openfaas
+spec:
+  domain: "example.matiaspan.dev"
+  function: "<username>-example-site"
+  ingressType: "nginx"
+  tls:
+    enabled: true
+    issuerRef:
+      name: "letsencrypt-prod"
+      kind: "Issuer"
+```
+* You need to update the `function` to match with the new one. OpenFaaS Cloud deploys function using the following convention: `<username>-<function name>`. If you are unsure about the name you can find it with `faas-cli list`.
+
+We won't be using our previous function anymore so remove it:
+```sh
+faas-cli remove example-site
+```
+
+Try creating a new blog post with: `hugo new posts/my-first-post.md`. Remember to run that command inside the folder of your hugo site, not the root folder of the project.  
+Commit and push your changes again, after OpenFaaS Cloud does its thing you will be able to see your new changes deployed. Awesome!
+
+## CI/CD with GitLab
+If you can't or don't want to install OpenFaaS Cloud but you still want to have a [Netlify](https://netlify.com)-like experience then you can host your repository on GitLab and use their free CI/CD offerings. Save the following to a YAML file in the root folder of the repo `.gitlab-ci.yml`:
+```yaml
+image: docker:stable
+
+stages:
+  - build
+
+services:
+   - docker:dind
+
+before_script:
+  - apk add --no-cache git
+  - if [ -f "./faas-cli" ] ; then cp ./faas-cli /usr/local/bin/faas-cli || 0 ; fi
+  - if [ ! -f "/usr/local/bin/faas-cli" ] ; then apk add --no-cache curl git && curl -sSL cli.openfaas.com | sh && chmod +x /usr/local/bin/faas-cli && /usr/local/bin/faas-cli template pull && cp /usr/local/bin/faas-cli ./faas-cli ; fi
+
+build:
+  stage: build
+  script:
+    - git submodule init && git submodule update
+    # Build Docker image
+    - /usr/local/bin/faas-cli template pull https://github.com/matipan/openfaas-hugo-template
+    - /usr/local/bin/faas-cli build --tag=sha --parallel=2 -f example-site.yml
+
+    # Login & Push Docker image to private repo
+    - echo -n "$CI_DOCKER_LOGIN" |  docker login --username $(echo -n "$CI_DOCKER_USERNAME") --password-stdin
+    - /usr/local/bin/faas-cli push --tag=sha -f example-site.yml
+    - echo -n "$CI_OF_PWD" | /usr/local/bin/faas-cli login --gateway $(echo -n "$CI_OF_GATEWAY") --username admin --password-stdin
+
+    # Deploy function from private repo
+    - /usr/local/bin/faas-cli deploy --tag=sha -f example-site.yml
+  only:
+    - master
```

Comment:
Was this adapted from the official docs? Link to it, if it was.

Reply:
It was, I'll add a link

Reply:
Did you add the link?